Mena_4pm_ProjectPortfolio_Team12.c

```diff
@@ -258,8 +261,22 @@ int CheckSpot(int row, int col, int board[][7]) {
     }
     return max;
 }
-int CheckWinner() {
-
+int CheckWinner(int board[][7], int winningSpace[]) {
+    int i, j;
+    for (i=0; i<6,i++) {
+        for (j=0;j<7;j++) {
+            if (checkSpot(board[i][j]>=4) {
```

Comment:
the function is CheckSpot not checkSpot and also the function has too few inputs. I found these out just by running the program on Geany btw, so I will try to find the right inputs as well for the function. It might help to remove CheckSpot from the for loop and set it equal to a variable first. Then use that variable in the for loop; essentially just simplify the for loop.